tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md

```diff
@@ -23,29 +23,29 @@ primary_tag: topic>abap-extensibility
 ## Intro
 You can get a generated UI for your custom business object and adapt it to your needs within the SAP S/4HANA Cloud tenant (in-app-extensibility). As building UIs in SAP BTP (side-by-side extensibility) offers more possibilities than in-app-extensibility, this tutorial gives you an idea how to build a UI based on a custom business object's OData service in SAP Business Application Studio, test it, and get it into the SAP S/4HANA Cloud tenant.
 
->Tutorial last updated in May 2021
+>Tutorial last updated with SAP S/4HANA Cloud Release 2302
 
 ---
 
 ### Launch SAP Business Application Studio
 
-SAP Business Application Studio is a subscription-based service in SAP BTP. The SAP trial account offers an easy access directly from <https://account.hanatrial.ondemand.com>, however the following sequence describes the procedure via the global account in SAP BTP cockpit as needed for customer accounts.
+**SAP Business Application Studio** is a subscription-based service in SAP BTP. The SAP trial account offers a Quick Tool Access directly from <https://account.hanatrial.ondemand.com>, however the following sequence describes the procedure via the global account in SAP BTP cockpit as needed for customer accounts. By choosing the trial specific **Quick Tool Access** (red dashed box) you can skip the following sequence.
 
-1.  In your web browser, open the SAP BTP Trial cockpit <https://account.hanatrial.ondemand.com> and **Enter Your Trial Account**, which is a so called global account.
+1.  In your web browser, open the SAP BTP Trial cockpit <https://account.hanatrial.ondemand.com> and **Go To Your Trial Account**, which is a so called global account.
 
-    <!-- border -->![Enter Global Trial Account ](btp-enter-global-trial-account.png)
+    ![Enter Global Trial Account ](btp-enter-global-trial-account-or-BAS.png)
 
 2.  On your global account page, select default subaccount `trial`.
 
-    <!-- border -->![Enter trial Subaccount](btp-enter-trial-subaccount.png)
+    ![Enter trial Subaccount](btp-enter-trial-subaccount.png)
 
 3.  In the navigation pane expand the **Services** section.
 
 4.  Select **Instances and Subscriptions**.
 
 5.  Click the link or the icon at the SAP Business Application Studio Subscription.
 
-<!-- border -->![Open SAP Business Application Studio from Subaccount](btp-open-bas.png)
+![Open SAP Business Application Studio from Subaccount](btp-open-bas.png)
```

Comment:
similar to the previous tutorial: I personally found the numbering on the screenshot a bit confusing. If I were reading the tutorial for the first time I would assume I screenshot with steps 1 and 2 is somehow missing. I would suggest to keep the numbering in the screenshot sequential (1,2,3) and maybe add it after every step, i.e:
"In the navigation pane expand the Services section (1)" and so on.

Reply:
left the numbering but put screenshot up to first step it shows